robots_101/programme_structure.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: page
+title: Robots 101 - Programme Structure
+---
+
+# Robots 101 - Programme Structure
+
+Student Robotics is an annual robotics competition for people aged 16-19.
+But what actually happens during the course of the competition year?
+
+## Registration
+
+Registration for the competition usually opens in late summer. Teams usually
```

Comment:
We try not to do things during summer holiday for teams
```suggestion
Registration for the competition usually opens in early September. Teams usually
```

Reply:
Hrm, I had actually thought we'd opened it earlier than this quite a few times but I can't find many examples. We do unfortunately end up opening in October quite a bit looking at the team-emails, though as we don't want to be doing that probably best not to advertise that.